src/js/slider.js

```diff
@@ -36,6 +38,7 @@
             var $handle = angular.element($element[0].querySelector('.ch-slider-handle'));
             var $fill = angular.element($element[0].querySelector('.ch-slider-fill'));
             var $line = angular.element($element[0].querySelector('.ch-slider-line'));
+            var $initValue = 0;
```

Comment:
The $ sign shows that the variable is a jQuery / angular element. So this should be `var initValue = 0;`